src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/service/UserService.kt

```diff
@@ -43,6 +45,9 @@ class UserService(
     } ?: throw UsernameNotFoundException("User $userId not found")
   }
 
+  suspend fun getAuthUserByUsername(username: String): User? =
+    userRepository.findByUsernameAndSource(StringUtils.upperCase(StringUtils.trim(username))) // ?: throw UsernameNotFoundException("Account for username $username not found")
```

Comment:
As earlier comment, i'd prefer to see the exception thrown here. But either way remove the commented code

Reply:
Moved to service